UnitsNet/Scripts/UnitDefinitions/ForceFlowRate.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+    "Name": "ForceFlowRate",
```

Comment:
I'm sorry, I really do appreciate your contributions, but this seems a lot like the PressureFlowRate case.

I can't seem to find much info on this unit on google and I'm neither familiar with it nor have an intuitive understanding of what force flow rate means.

Do you consider this unit more widely used or accepted than pressure flow rate?

However, MassFlowRate, which you originally added, does seem like a thing. So KilogramPerSecond, PoundPerSecond, TonnePerSecond and so on all seem like widely used units and I have an intuitive understanding of what it means. 

Did you intentionally remove MassFlowRate or is that something you still want to add?